src/renderer/pages/Project.tsx

```diff
@@ -117,56 +118,63 @@ const ProjectPage = () => {
                   <LeftAligned
                     style={{ marginLeft: '20px', marginRight: '20px' }}
                   >
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={returnToHome}
-                      title="Exit to Home"
-                      style={{ marginRight: '25px' }}
-                    >
-                      <HomeIcon />
-                    </IconButton>
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={performUndo}
-                      title="Undo"
-                    >
-                      <UndoIcon />
-                    </IconButton>
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={performRedo}
-                      title="Redo"
-                    >
-                      <RedoIcon />
-                    </IconButton>
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={deleteText}
-                      title="Delete Text"
-                    >
-                      <DeleteIcon />
-                    </IconButton>
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={cutText}
-                      title="Cut Text"
-                    >
-                      <ContentCutIcon />
-                    </IconButton>
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={copyText}
-                      title="Copy Text"
-                    >
-                      <ContentCopyIcon />
-                    </IconButton>
-                    <IconButton
-                      sx={{ color: colors.grey[300] }}
-                      onClick={pasteText}
-                      title="Paste Text"
-                    >
-                      <ContentPasteIcon />
-                    </IconButton>
+                    <ProjectTooltip title="Exit to Home">
```

Comment:
This whole list is pretty repetitive, could we just have a list of objects containing title, action and icon and then render the buttons by mapping over the list? Or at least a helper function to generate a button given a tooltip title, action and icon